### PR TITLE
Second stage

### PR DESCRIPTION
**Pull request by anajov**
MessageDao class used for DB access is added. Tests added.
GET recent messages supported.

### CODE REVIEW

File: docker/db/db_init.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (nealstewart) [inline docker/db/db_init.sh]: SQL in bash file :hear_no_evil:

Comment (nealstewart) [inline docker/db/db_init.sh]: you don't need to change anything :see_no_evil:

Comment (anajov) [inline docker/db/db_init.sh]: so to confirm - no need to change anything ? :D

File: src/main/java/com/tradeshift/messages/service/MessageService.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public static final String NO_RECENT_MESSAGES = "No recent messages";
+    private final MessageDao msgDAO;
+
+    @Autowired
+    public MessageService(MessageDao msgDAO) {
+        this.msgDAO = msgDAO;
+    }
+
+    public ResponseMessage getResponseMessage(String name) {
+        msgDAO.insertMessage(name);
```
Comment (nealstewart) [inline src/main/java/com/tradeshift/messages/service/MessageService.java]: Your DAO should return the DTO that was created.

File: src/main/java/com/tradeshift/messages/service/MessageService.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        return new ResponseMessage(name);
```
Comment (nealstewart) [inline src/main/java/com/tradeshift/messages/service/MessageService.java]: Then this entire method can just be `return msgDAO.insertMessage`

File: src/main/java/com/tradeshift/messages/service/MessageService.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import org.joda.time.format.ISODateTimeFormat;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by ajo on 08/06/15.
```
Comment (nealstewart) [inline src/main/java/com/tradeshift/messages/service/MessageService.java]: Don't include the "Created by" snippets. Git will tell people who made the file :)

**Comment by nealstewart** [comment]
:game_die: No need to fix comments, just read them and be aware in future

**Pull request merged**